.github/workflows/macos.yml

```diff
@@ -34,19 +34,21 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: generate-lockfile
-      - name: Cache cargo dirs
-        uses: actions/cache@v2
-        with:
-          path: |
-            ~/.cargo/registry
-            ~/.cargo/git
-            ~/.cargo/bin
-          key: ${{ matrix.version }}-x86_64-apple-darwin-cargo-${{ hashFiles('**/Cargo.lock') }}
-      - name: Cache cargo build
-        uses: actions/cache@v2
-        with:
-          path: target
-          key: ${{ matrix.version }}-x86_64-apple-darwin-cargo-build-trimmed-${{ hashFiles('**/Cargo.lock') }}
+
+      # - name: Cache cargo dirs
```

Comment:
Is there a reason that these are commented out?

Reply:
Yeah the macos build was thowing nonsense build errors and figured it was cache artifacts related.

Reply:
I think then it makes sense to add a comment why these lines were commented. Commented code without an explanation usually raises questions.